### PR TITLE
Russian translate

### PR DESCRIPTION
**Pull request by krotesk**
Corrected Russian translation

### CODE REVIEW

File: packages/tools/locales/ru_RU.po [hunk 1]
```diff
@@ -1649,16 +1651,15 @@ msgstr "Включить мягкие отступы"
 
 #: packages/lib/models/Setting.ts:1048
 msgid "Enable spellcheck in the text editor"
-msgstr ""
+msgstr "Включите проверку орфографии в текстовом редакторе"
```
Comment (roman-r-m) [inline packages/tools/locales/ru_RU.po]: Может "включить" лучше?

**Review by tessus — COMMENTED** [review]
Thanks, the PR looks good. I can't say anything to Roman's comment. I'll merge as soon as it is resolved.

File: packages/tools/locales/ru_RU.po (hunk 1 above)
Comment (krotesk) [reply packages/tools/locales/ru_RU.po]: Поправил

**Comment by tessus** [comment]
Thanks for the update!

**Pull request merged**